inference-engine/src/gna_plugin/gna_graph_compiler.cpp

```diff
@@ -112,6 +112,12 @@ void GNAGraphCompiler::fillConcatConnections(InferenceEngine::CNNLayerPtr layer)
                 InferenceEngine::details::product(begin(dataInput->getDims()),
                                                   end(dataInput->getDims())) * dataInput->getPrecision().size();
 
+        // concat align layer can have additional padding, so the size of layer needs to be calculated
+        // based on original number of rows
+        if (ptrConcatLayerInput->CheckParamPresence("original_num_rows")) {
```

Comment:
@esmirno please take a look at this change and let me know if it is acceptable.

Reply:
this looks OK